pkg/client/injection/reconciler/serving/v1alpha1/knativeserving/stub/controller.go

```diff
@@ -0,0 +1,54 @@
+/*
+Copyright 2020 The Knative Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// Code generated by injection-gen. DO NOT EDIT.
+
+package knativeserving
+
+import (
+	context "context"
+
+	configmap "knative.dev/pkg/configmap"
+	controller "knative.dev/pkg/controller"
+	logging "knative.dev/pkg/logging"
+	knativeserving "knative.dev/serving-operator/pkg/client/injection/informers/serving/v1alpha1/knativeserving"
+	v1alpha1knativeserving "knative.dev/serving-operator/pkg/client/injection/reconciler/serving/v1alpha1/knativeserving"
+)
+
+// TODO: PLEASE COPY AND MODIFY THIS FILE AS A STARTING POINT
```

Comment:
Delete

Also, is the code generated comment still relevant? If so, we might want to remove this from the generated code :)

Reply:
In serving, the content under stub is autogenerated  and checked in: https://github.com/knative/serving/tree/fdfc49f209d6114295968d9d4c7ad8392542a9c0/pkg/client/injection/reconciler/serving/v1/service/stub